easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.1-gcccuda-2016.10.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.0.1'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'gcccuda', 'version': '2016.10.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads',]
+
+#patches = [
+#    'OpenMPI-%(version)s_common-cuda-lib.patch',
+#    'OpenMPI-%(version)s-vt_cupti_events.patch',
+#]
```

Comment:
Maybe I didn't look at the patches properly. I'll do that and see if they are still needed or not.